.ci/checker-framework.groovy

```diff
@@ -443,68 +439,18 @@ class CheckerFrameworkError implements Comparable<CheckerFrameworkError> {
             return i
         }
 
-        if (this.isUnstable() || other.isUnstable()) {
```

Comment:
Why is `compareTo` changing when this says it is just changing equals/hash? The 2 are not connected together so I don't understand why we are including them together.

Reply:
`compareTo` is also incorrect, we do not need any special comparison when an error is `unstable`, added it to the commit message and PR title.

Reply:
I am seeing a lot more changing than unstable. Message and details changed. I assume this is to rewriting `temp-var` and `capture`?

Please put all changes to first post as well since you are documenting this as your issue.

Reply:
Done.